client/src/shared/Constants.ts

```diff
@@ -15,7 +16,7 @@ export const CONTENT_WIDTH = 1100;
 
 /* API */
 export const API_BASE =
-  process.env.NODE_ENV === 'development' ? 'https://ug6g4yngl0.execute-api.us-east-1.amazonaws.com/test' : 'https://ug6g4yngl0.execute-api.us-east-1.amazonaws.com/test';
+  process.env.NODE_ENV === 'development' ? 'https://iqlbtpcb4g.execute-api.us-west-2.amazonaws.com/Prod' : 'https://iqlbtpcb4g.execute-api.us-west-2.amazonaws.com/Prod';
```

Comment:
I realized that this is a bit awkward, because we bind one API gateway with one Dynamo function, which can handle only a single index, but not multiple (e.g., both acl and cord19).

Maybe we can, in our cloudformation/sam template, declare two search lambda functions and map one API to the two functions? E.g., have the `acl` function listening on path `<API_BASE>/<ENV>/acl/search` and have the `cord19` function listening on path `<API_BASE>/<ENV>/cord19/search`.

@shaneding wdyt?